packages/editor-ui/src/views/WorkflowOnboardingView.vue

```diff
@@ -26,10 +27,10 @@ const openWorkflowTemplate = async (templateId: string) => {
 			interpolate: { name: template.name },
 		});
 
-		const workflow = await workfowStore.createNewWorkflow({
+		const workflow = await workflowsStore.createNewWorkflow({
 			name,
 			connections: template.workflow.connections,
-			nodes: template.workflow.nodes,
+			nodes: template.workflow.nodes as INode[],
```

Comment:
Is there a way to avoid typecasting here?

Reply:
done!